src/screens/CreateTest/CreateQuestion/CreateAnswer/index.tsx

```diff
@@ -43,10 +47,12 @@ export const CreateAnswer = ({answers}: CreateAnswerPropsType) => {
   return (
     <View>
       <TextBox>Answer choice</TextBox>
-      {answers.map(item => (
+      {answers.map((item, index) => (
         <AddingAnswer
-          key={item.id}
+          key={index}
```

Comment:
https://reactjs.org/docs/lists-and-keys.html#:~:text=We%20don%E2%80%99t%20recommend,indexes%20as%20keys.